### DISCUSSION TITLE
Full screen mirroring

### DISCUSSION THREAD

**Discussion by marcofrancis** [blinksh/blink#2041] [Support]
I’m running version 17.3 on a non-M1 ipad (so no full stage manager). Is there a way to have the external monitor mirror the ipad without having the black lateral bars? I need to mirror because I’d like to use vscode on the external monitor and would like to use the mouse/trackpad with it. Also is there any doc that explains what the different options for the external monitor mean and how to move focus from the ipad screen to the external monitor?

**Comment by carloscabanero** [comment]
When you "mirror" the iPad, the resolution is set to that of the iPad as it is "mirroring" the device. The resolution is usually a 4:3 or close, while external displays are around 16:9, and hence the bars. It isn't possible to change this behavior with "mirroring".

There is no article at the moment describing the modes, but let me mark  but the additional mode you have available in your device is "Projection". Projection gives you a full remote terminal on the external display. You can use this mode also with Apple TV for presentations, even without connecting your device. This terminal will have the same resolution as the external display (so there would be no black bars). You can switch between the terminals on your device and the external display using cmd-shift-O (the shortcut can be changed in config > keyboard > blink shortcuts). The problem is this external terminal doesn't have mouse control (this is a limitation on iOS).
